internal/datasource/os/exec/exec.go

```diff
@@ -13,9 +13,7 @@ import (
 )
 
 //go:generate go run github.com/maxbrunsfeld/counterfeiter/v6@v6.7.0 -generate
-//counterfeiter:generate -o mock_exec.go . ExecInterface
-//go:generate sh -c "grep -v github.com/nginx/agent/v3/internal/datasource/os/exec mock_exec.go | sed -e s\\/exec\\\\.\\/\\/g > mock_exec_fixed.go"
-//go:generate mv mock_exec_fixed.go mock_exec.go
+//counterfeiter:generate . ExecInterface
 type ExecInterface interface {
```

Comment:
can you add tests for exec.go?

Reply:
Don't know how I can test this one. Do you know of a way of mocking the `os/exec` package?

Reply:
does this work? https://npf.io/2015/06/testing-exec-command/